includes/cf/sonarr-streaming-services.md

```diff
@@ -17,3 +17,4 @@
     | [{{ sonarr['cf']['pcok']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#pcok)     |  {{ sonarr['cf']['pcok']['trash_scores']['default'] }}  | {{ sonarr['cf']['pcok']['trash_id'] }}   |
     | [{{ sonarr['cf']['sho']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#sho)       |  {{ sonarr['cf']['sho']['trash_scores']['default'] }}   | {{ sonarr['cf']['sho']['trash_id'] }}    |
     | [{{ sonarr['cf']['stan']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#stan)     |  {{ sonarr['cf']['stan']['trash_scores']['default'] }}  | {{ sonarr['cf']['stan']['trash_id'] }}   |
+    | [{{ sonarr['cf']['syfy']['name'] }}](/Sonarr/sonarr-collection-of-custom-formats/#syfy)     |  0  | {{ sonarr['cf']['syfy']['trash_id'] }}   |
```

Comment:
Thought we were doing dynamic scores for all for maintainability

Reply:
We are indeed